Sources/Proxy/Core/ViewProxy/index.js

```diff
@@ -257,8 +257,12 @@ function vtkViewProxy(publicAPI, model) {
 
   // --------------------------------------------------------------------------
 
+  publicAPI.lightResetCamera = model.renderer.resetCamera;
```

Comment:
please add documentation. it is not clear from the name what this lightResetCamera do